Classes/Network/KalturaPlaybackRequestAdapter.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  KalturaPlaybackRequestAdapter.swift
+//  Pods
+//
+//  Created by Gal Orlanczyk on 05/04/2017.
+//
+//
+
+import Foundation
+
+class KalturaPlaybackRequestAdapter: PKRequestParamsAdapter {
+    
+    private var playSessionId: UUID
+    
+    init(playSessionId: UUID) {
+        self.playSessionId = playSessionId
+    }
+    
+    public static func setup(player: Player) {
+        let adapter = KalturaPlaybackRequestAdapter(playSessionId: player.sessionId)
+        player.settings.set(contentRequestAdapter: adapter)
+    }
+    
+    public func adapt(requestParams: PKRequestParams) -> PKRequestParams {
+        guard requestParams.url.path.contains("/playManifest/") else { return requestParams }
```

Comment:
The actions are case-sensitive. It doesn't work in lower-case. This doesn't work (404):
https://cdnapisec.kaltura.com/p/1851571/playmanifest/entryId/0_pl5lbfo0/format/url/protocol/https/a.mp4
This does:
https://cdnapisec.kaltura.com/p/1851571/playManifest/entryId/0_pl5lbfo0/format/url/protocol/https/a.mp4